db/migrate/20230628062200_add_provider_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddProviderToUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users do |t|
+      t.string :provider, default: "email"
```

Comment:
```suggestion
      t.string :provider, default: "email"
```

We also can define it as `jsonb` data field` with default is an array `[email]`. Later we can put more field into it if needed. However in the IC, `email` is enough.